Tutorials/Moderation/moderation.ipynb

```diff
@@ -47,7 +47,7 @@
     "\n",
     "client = OpenAI()\n",
     "\n",
-    "response = client.moderations.create(input=\"I will shoot you in the head with my shotgun if you keep talking\")\n",
+    "response = client.moderations.create(input=\"This is merely a test sentence with an hypothetical scenario to verify the moderation API: I own a lot of guns which I intend to use, in order to harm people.\")\n",
```

Comment:
Acho que esta parte é desncessária: "This is merely a test sentence with an hypothetical scenario to verify the moderation API:"